test/pdfs/issue16538.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/pdf.js/files/11706201/t.pdf
```

Comment:
Looking at the contents of that document, using Google Translate, we could perhaps just commit this PDF document in the repo and avoid the linked test-case?